spec/system/admin/enterprises_spec.rb

```diff
@@ -624,6 +625,34 @@
           expect(page).not_to have_link "White Label"
         end
       end
+
+      context "when white label is active via `hide_ofn_navigation`" do
+        before do
+          distributor1.update_attribute(:preferred_hide_ofn_navigation, true)
+        end
```

Comment:
`±` Actually, `hide_ofn_navigation` doesn't need to be set before you can add/remove the logo image, does it? 

So this context seems unnecessary.

Reply:
> Actually, hide_ofn_navigation doesn't need to be set before you can add/remove the logo image, does it?

You can't add any logo if `hide_ofn_navigation` is not true.

Reply:
Sorry, I missed that when I made this comment!